app/scenes/Login/Login.js

```diff
@@ -1,20 +1,69 @@
 import {
   Text,
-  View
+  View,
+  ToastAndroid
 } from 'react-native';
+import LoginView from './LoginView';
+import message from '../../lang/en';
+import uri from '../../config/uri';
 import React, {Component} from 'react';
+import {httpUtils, validationUtils} from '../../utils';
 
 import styles from './styles';
 
 class Login extends Component {
 
+    constructor() {
+        super();
+        this.state = {
+          login: {
+            username: '',
+            password:''
+          }
+         };
+    }
+
   render() {
     return (
       <View style={styles.container}>
-        <Text>Login</Text>
+        <LoginView
+          data={this.state.login}
+          onPress={this._loginUser}
+          handleOnChange={this._handleOnChange}
+        />
       </View>
     );
   }
+
+  /**
+   * Set user input in state.
+   * @param {string} path
+   * @param {object} value
+   * @private
+  */
+  _handleOnChange = (path, value) => {
+    let [object, key] = path.split('.');
+    this.state[object][key] = value;
+
+    this.setState({[object]: this.state[object]});
+  };
+
+  /**
+  * Login user and on success redirect to dashboard.
+  * @private
+  */
+  _loginUser = () => {
+    if (!validationUtils.isEmailValid(this.state.login.username)) {
+      ToastAndroid.show(message.WRONG_EMAIL_FORMAT, ToastAndroid.SHORT);
```

Comment:
Since we are target is for cross platform (android and ios) so let's don't use ToastAndroid